src/cloud/live/site-launch-checklist.md

```diff
@@ -16,51 +16,80 @@ Before you deploy to the Production environment, download the [Launch checklist]
 
-After checking with your registrar about where to change your DNS settings, add a CNAME record for your website that points to the Fastly service: `prod.magentocloud.map.fastly.net`. If you use multiple hostnames for your site, you must add a CNAME record for each one.
+When you are ready to go live, you must update the DNS configuration to route traffic from your Production environment through the Fastly service.
 
- {:.bs-callout-info}
-This does not work for an [apex domain](https://blog.cloudflare.com/zone-apex-naked-domain-root-domain-cname-supp) (also referred to as a _naked_ domain). You must use a DNS provider that supports forwarding DNS queries to use an apex domain.
+**Prerequisites:**
 
-The following list contains examples of DNS providers for informational purposes. Use your preferred DNS provider.
+-  [Set up and test Fastly in your development environment]({{ site.baseurl }}/cloud/cdn/configure-fastly.html)
 
--  CNAME with ALIAS record from [Dyn](http://dyn.com)
--  ANAME record on [DNS Made Easy](http://www.dnsmadeeasy.com)
--  ANAME at [easyDNS](https://www.easydns.com)
--  ACNAME at [CloudFlare](https://www.cloudflare.com)
--  ALIAS at [PointDNS](https://pointhq.com)
+-  Production environment configuration has been updated with all required domains
 
-Many other DNS providers also offer workarounds to accomplish this goal. The most common is to add a CNAME record for the `www` host on the domain and then use the DNS provider's redirect service to redirect the apex over to the `www` version of the domain. Consult your DNS provider for more information.
+   Typically, you work with your Customer Technical Advisor to add all top level domains and sub-domains required for your Magento stores. To add or change the domains for your Production environment, submit a [Magento Support ticket](https://support.magento.com/hc/en-us/articles/360019088251). Wait for confirmation that your project configuration has been updated.
 
-Another option for apex domain is to add A records, which maps a domain name to the Fastly IP addresses:
+   On Starter projects, you must add the domains to your project. See [Manage domains]({{ site.baseurl }}/cloud/cdn/configure-fastly-customize-cache.html#manage-domains).
 
--  `151.101.1.124`
--  `151.101.65.124`
--  `151.101.129.124`
--  `151.101.193.124`
+-  SSL/TLS certificate provisioned for your production environments.
 
-### TLS and Fastly
+   If you added the ACME challenge records for your Production domains during the Fastly setup process, Magento has already validated the domains and provisioned the certificate.  The certificate will be applied to your Production environment automatically when you update the DNS configuration to point your Production domains to the Fastly service. If you did not pre-provision the certificate, or if you updated your domains, Magento must complete domain validation and provision the certificate, which takes up to 12 hours.
```

Comment:
The will be is making my eye twitch, but I don't have any good suggestion. I did fix an errant space, though.

```suggestion
   If you added the ACME challenge records for your Production domains during the Fastly setup process, Magento has already validated the domains and provisioned the certificate. The certificate will be applied to your Production environment automatically when you update the DNS configuration to point your Production domains to the Fastly service. If you did not pre-provision the certificate, or if you updated your domains, Magento must complete domain validation and provision the certificate, which takes up to 12 hours.
```